packages/app-store/zoho-bigin/lib/CrmService.ts

```diff
@@ -141,16 +140,18 @@ export default class BiginCalendarService implements Calendar {
       },
       data: JSON.stringify({ data: contacts }),
     });
+
+    return response.data;
```

Comment:
Also, I am thinking that we should replace all usages of axiom in AppStore to our own wrapper of Axiom that would make the `data` as `unknown` instead of `any`. This would fail TS check which is passing right now.

Reply:
I'm not a huge fan of using axiom in our app store. What if we just followed up with replacing instances of it with `fetch`?

Reply:
Yeah we could do that as an enhancement. Easy to pick by a contributor.